x-pack/test/api_integration/apis/logs_ui/log_threshold_alert.ts

```diff
@@ -125,6 +138,7 @@ export default function ({ getService }: FtrProviderContext) {
               },
             ],
           ]);
+          expect(alertLimit.setLimitReached.calledOnceWith(true)).to.be(true);
```

Comment:
Similar situation here. `with group by should work` doesn't tell me anything about the expected limit behavior.